langchain4j-mongodb-atlas/src/test/java/dev/langchain4j/store/embedding/mongodb/MongoDbEmbeddingStoreMiscIT.java

```diff
@@ -0,0 +1,122 @@
+package dev.langchain4j.store.embedding.mongodb;
+
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.model.Filters;
+import dev.langchain4j.data.document.Metadata;
+import dev.langchain4j.data.embedding.Embedding;
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import dev.langchain4j.store.embedding.EmbeddingMatch;
+import dev.langchain4j.store.embedding.EmbeddingSearchRequest;
+import dev.langchain4j.store.embedding.EmbeddingStore;
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.mongodb.MongoDBAtlasLocalContainer;
+
+import java.time.Duration;
+import java.util.List;
+
+import static dev.langchain4j.store.embedding.TestUtils.awaitUntilAsserted;
+import static dev.langchain4j.store.embedding.mongodb.MongoDbTestFixture.*;
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatNoException;
+import static org.assertj.core.data.Percentage.withPercentage;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+class MongoDbEmbeddingStoreMiscIT {
+
+    public static class ContainerIT extends MongoDbEmbeddingStoreMiscIT {
+        static MongoDBAtlasLocalContainer mongodb = new MongoDBAtlasLocalContainer("mongodb/mongodb-atlas-local:7.0.9");
+
+        @BeforeAll
+        static void start() {
+            MongoDbTestFixture.assertDoContainerTests();
+            mongodb.start();
```

Comment:
This should be done once in static block

Reply:
Addressed as [described](https://github.com/langchain4j/langchain4j/pull/1932#discussion_r1805103259).